### PR TITLE
SPEARBIT-168 Consistent operations sequence in `skim`

### PR DESCRIPTION
**Pull request by dglowinski**

### CODE REVIEW

File: src/EVault/modules/Vault.sol [hunk 1]
```diff
@@ -191,8 +191,9 @@ abstract contract VaultModule is IVault, Base, AssetTransfers, BalanceUtils {
         Shares shares = assets.toSharesDown(vaultCache);
         if (shares.isZero()) revert E_ZeroShares();
 
-        increaseBalance(vaultCache, receiver, account, shares, assets);
+        // skim performs equivalent order of operations as the deposit function, without assets being pulled from the user
```
Comment (hoytech) [inline src/EVault/modules/Vault.sol:194]: Let's kill this comment, I don't understand why anyone would care about this and it's just confusing.

Comment (kasperpawlowski) [reply src/EVault/modules/Vault.sol:194]: well, it was auditor's suggestion to have it... :)

Comment (hoytech) [reply src/EVault/modules/Vault.sol:194]: Yeah but it's more confusing than helpful IMO

Comment (dglowinski) [reply src/EVault/modules/Vault.sol]: comment removed

**Review by hoytech — APPROVED** [review]

**Pull request merged**